test-case/check-playback.sh

```diff
@@ -88,6 +94,14 @@ do
             do
                 dlogi "Testing: (Round: $round/$round_cnt) (PCM: $pcm [$dev]<$type>) (Loop: $i/$loop_cnt)"
                 dlogc "aplay -D$dev -r $rate -c $channel -f $fmt_elem -d $duration $file -v -q"
+                #sleep $random_min ~$ramdom_max seconds before running aplay, then aplay can be run under dsp active or suspended status randomly
+                if [ ${OPT_VALUE_lst['R']} -eq 1 ]; then
+                    time=$(func_lib_get_random $random_min $random_max)
+                else
+                    time=$random_min
+                fi
+                dlogi "Sleep: $time second(s)"
+                sleep $time
```

Comment:
The sleep should go after the aplay finished.
Do not inject any other cmd between dlogc and the real cmd, it will make log more confusing.